targets/TARGET_NXP/TARGET_LPC176X/device/TOOLCHAIN_GCC_ARM/TARGET_XPRESSO_LPC1769/XPRESSO_LPC1769.ld

```diff
@@ -0,0 +1,172 @@
+/* Linker script for mbed LPC1769 */
```

Comment:
Is this file required? Should it not use the LPC1768 one? It looks like it may have come from the xbed port unnecessarily?

If it is required please can you update it to to include my recent changes to LPC1768 regarding CRP and bootloader as in #5038.